src/library_dylink.js

```diff
@@ -779,6 +781,29 @@ var LibraryDylink = {
             start = HEAPU8.indexOf(0, start) + 1;
           }
         }
+
+        function addEmJs(name, sig, body) {
+          sig = sig.slice(1, -1).split(',');
+          var args = [];
+          for (var i in sig) {
+            args.push(sig[i].split(' ').pop());
+          }
```

Comment:
Maybe add some comments here? I'm having a hard time following what this is doing. Looks like it's parsing arguments? But why is it doing this split-pop operation?